ragna/deploy/_ui/styles.py

```diff
@@ -1,155 +1,86 @@
+    "global": [pn.widgets.TextInput, pn.widgets.Select, pn.widgets.Button],
+    "source_accordion": [
+        pn.layout.Accordion,
+        pn.layout.Card,
+        pn.pane.HTML,
+        pn.pane.Markdown,
+    ],
+    "chat_info": [pn.pane.Markdown, pn.widgets.Button],
+    "auth": [pn.widgets.TextInput, pn.pane.HTML, pn.widgets.Button, pn.Column],
+    "central_view": [pn.Column, pn.Row, pn.pane.HTML],
+    "chat_interface": [
+        pn.widgets.TextInput,
+        pn.layout.Card,
+        pn.pane.Markdown,
+        pn.widgets.button.Button,
+        pn.Column,
+    ],
+    "right_sidebar": [pn.widgets.Button, pn.Column, pn.pane.Markdown],
+    "left_sidebar": [pn.widgets.Button, pn.pane.HTML, pn.Column],
+    "main_page": [pn.Row],
+    "modal_welcome": [pn.widgets.Button],
+    "modal_configuration": [
+        pn.widgets.IntSlider,
+        pn.layout.Card,
+        pn.Row,
+        pn.widgets.Button,
+    ],
+}
 
-                        /* Resets some existing styles */
-                        :host(.accordion) { 
-                            margin-left: 0px;
-                            margin-top: 0px;
-                            outline: none;
-                        }
-                        
-                        /* Styles the button itself */
-                        button.accordion-header { 
-                            background-color: white !important;
-                            height: var(--ragna-accordion-header-height);
-                            padding-top: 0px;
-                            padding-bottom: 0px;
-                            outline:0px;
-                            margin-left: 15px;
-                            margin-right: 15px;
-                            width: calc(100% - 30px);
-                            border-bottom: 2px solid #D9D9D9;
-                        
-                        }
-                        
-                        button.accordion-header div.card-button {
-                            font-size: 11px;
-                            padding-top: 5px;
-                            margin-left: 0px;
-                            margin-right: 10px;
-                        }
-                        
-                        div.card-header-row {
-                            height: var(--ragna-accordion-header-height);
-                            background-color: unset !important;
-                        }
 
-                        /* styles the content of the sources content (the expanding areas of the Accordion) */
-                        div.bk-panel-models-markup-HTML.markdown {
-                            margin-left: 15px;
-                            margin-right: 15px;
-                            margin-top:0px;
-                        }
+def apply_design_modifiers():
+    css_filepaths = []
+    for dir, classes in css_modifiers.items():
+        for cls in classes:
+            css_filename = cls.__name__.lower().split(".")[-1] + ".css"
```

Comment:
A class name can never have a period `.` in it. Should this just be

```suggestion
            css_filename = f"{cls.__name__.lower()}.css"
```

?